app/code/Magento/Cms/Test/Unit/Model/BlockTest.php

```diff
@@ -0,0 +1,342 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
```

Comment:
We cannot use `strict_types` for Magento 2.1 since it's compatible with PHP 5.6. The strict_types introduced in PHP 7. Please, remove the declaration